docs/source/jupyter/cuds-api.ipynb

```diff
@@ -1,295 +1,699 @@
+        "\n",
+        "The most important functionalities that the CUDS data structure exposes as python methods are the following:\n",
+        "- **add**: Connects the current CUDS object to one or more CUDS objects through a specific ontology relationship.\n",
+        "- **remove**: Despite its name, it does NOT delete the CUDS object itself. Instead, it just disconnects the current CUDS object from one or more CUDS objects.\n",
+        "- **get**: Returns the CUDS objects connected to the current CUDS object through a specific ontology relationship. \n",
+        "- **iter**: Similar to the `get` method, it just returns one CUDS objects at a time instead of all at once (python iterator), so that memory can be saved.\n",
+        "- **is_a**: Checks if the CUDS object is an instance of the given oclass.\n",
+        "\n",
+        "In addition, other important functionalities are exposed as python properties:\n",
+        "- **oclass**: The ontology class of the ontology individual represented by the CUDS object. If the individual belongs to multiple classes, only one of them is referenced by the property.\n",
+        "- **uid**: A unique ID identifying an ontology individual.\n",
+        "- **iri**: The [Internationalized Resource Identifier](https://fusion.cs.uni-jena.de/fusion/blog/2016/11/18/iri-uri-url-urn-and-their-differences/) of the CUDS object. It consists of a CUDS namespace prefix and the unique ID of the CUDS object. This will be clarified in the tutorial.\n",
+        "- **attributes**: The values of the ontology attributes of an individual (also known as [data properties](https://www.w3.org/TR/owl2-syntax/#Data_Properties) in [OWL](https://en.wikipedia.org/wiki/Web_Ontology_Language)) may also be accessed as python properties of the CUDS objects. For example: `cuds_object.name`.\n",
+        "\n",
+        "There are some advanced functionalities NOT covered in this tutorial. Among them, we highlight the **update** method, which is covered in the [wrapper tutorial](multiple-wrappers.ipynb), where it can be seen in action. For a complete list of available methods and properties, check the API reference. That whole set of methods and attributes constitutes the CUDS [API](https://en.wikipedia.org/wiki/API).\n",
+        "\n",
+        "[//]: # (TODO: Link to API reference.)"
+      ],
+      "metadata": {}
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Let's get hands on\n",
+        "In this tutorial, we will work with the `city` namespace, the example namespace from OSP-core. It consists of concepts from the example [city ontology](https://github.com/simphony/osp-core/blob/master/osp/core/ontology/docs/city.ontology.yml). There is [another tutorial](#) on the ontology YAML installation file. However, it is not important for the purposes of this tutorial.\n",
```

Comment:
I just noticed this and removed it in the new commit. Do we even have a YAML tutorial? [What we have](https://simphony.readthedocs.io/en/latest/yaml.html) just looks like a normal documentation section (that I could link to as well).

Reply:
Notice that in the [current version](https://simphony.readthedocs.io/en/latest/jupyter/cuds-api.html) of the docs it is also empty.

Reply:
It was supposed to be done at some point, but I guess it never had enough priority. Maybe linking to the normal documentation would suffice.

Reply:
I linked to the ontology installation documentation. It is a quite long sentence, but I think that since we do not have the tutorial, it's necessary to link to those three spots. 

As an alternative we could just remove it by reverting to [6bd48b1](https://github.com/simphony/docs/pull/102/commits/6bd48b18ebc8021526cac643f92dc1ce03c27bb7).